### ISSUE TITLE
Object Marker: Buttons as highlighted in red are not reachable via keyboard (no visible focus provided).

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3821]
Description: Buttons as highlighted in red are not reachable via keyboard (no visible focus provided).

Expected:  Ideally each and every UI element should be reachable and usable to the user via keyboard.

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/98773355-c64cdc00-240e-11eb-9c73-b77f54bd9798.png)

![image](https://user-images.githubusercontent.com/32538291/98773365-ca78f980-240e-11eb-8b97-61facc416e2d.png)

**Comment by Lokanathan-k** [comment]
closed as part of this PR https://github.com/SAP/fundamental-ngx/pull/4081
